lib/internal/bootstrap_node.js

```diff
@@ -270,7 +270,12 @@
 
   function installInspectorConsole(globalConsole) {
     const wrappedConsole = NativeModule.require('console');
-    const inspector = process.binding('inspector');
+    var inspector;
+    try {
+      inspector = process.binding('inspector');
+    } catch (ignored) {
+      return wrappedConsole;
+    }
     const config = {};
     for (const key of Object.keys(wrappedConsole)) {
       if (!globalConsole.hasOwnProperty(key))
```

Comment:
Yep, that is correct. I'm not sure now what I was thinking as that sounds much more sensible (or if I ran into some different issue doing that) but I'll take a closer look at this tomorrow.